Portfolio/index.css

```diff
@@ -1,174 +1,123 @@
-console.log('Hello World');
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;
-}
-/* Additional CSS styles */
-.exp img{
 
-width: 200px;
-height: 200px;
-border: 2px solid black;
-border-radius: 100px;
 
-}
 
-/* Header styling */
-header {
-background-color: #ffffff;
-color: rgb(0, 0, 0);
-text-align: center;
-padding: 20px 0;
+.navbar img {
+    margin-left: 10px;
 }
 
-/* Section styling */
-section {
-background-color: #ffffff;
-padding: 20px;
-margin: 20px;
-border-radius: 8px;
-box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-}
+body {
 
-/* Skills section styling */
-.skills {
-display: flex;
-flex-direction: column;
-align-items: center;
+    background-color: #a9b5b8;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: rgb(79, 77, 77);
+    margin: 0;
+    padding: 0;
 }
 
-.skills h2 {
-color: rgb(0, 0, 0);
-margin-bottom: 20px;
+.abtme h3,
+h4 {
+    margin-left: 10px;
 }
 
-.skills ul {
-list-style-type: none;
-padding: 0;
-margin: 0;
+.btn button {
+    border: 1px solid;
+    border-radius: 10px;
+    cursor: pointer;
+    background: transparent;
+    padding: 6px 8px;
+    margin-left: 10px;
 }
 
-.skills ul li {
-color: rgb(0, 0, 0);
-margin-bottom: 10px;
-}
+.skills img {

```

Comment:
The closing curly brace for the '.last' selector is missing at the beginning of line 76.